### DISCUSSION TITLE
how do i make a video with black bars all around fit my screen?

### DISCUSSION THREAD

**Discussion by ddw04** [tamius-han/ultrawidify#207] [Q&A]
![Untitled](https://user-images.githubusercontent.com/77309593/205898056-058ee2c1-19ea-47e1-bcae-67188b46bdb8.png)
so on amazon or netflix when theres a movie/show that, before pressing full screen, is a tiny square with black bars on all sides goes full screen i cant seem to fit height for my monitor and im not sure why. ive tried pressing all the shorcut keys just too see if something happens and it doesnt. i tried to do custom aspect ratio, but nothing saves. help? im on v5.1.5

**Comment by tamius-han** [comment]
Chances are player element is not being detected correctly. In extension popup, go to 'advanced settings'. There, try enabling 'manually specify player' and then increasing "player is the n-th parent of video" box. Start with 1, save, reload page, press 'w' in full screen to see if it works. If it doesn't, increase the number by 1 until it does. 

![Screenshot_20221206_221800](https://user-images.githubusercontent.com/12505802/206025187-0ca26439-b937-4bad-b64e-49e407572e4e.png)

Normally, you should get something that works before that number goes over 10, and certainly before it reaches 15-20. 

If you see a `Query selectors for player:` input instead of the `Player is the n-th parent` input, click 'use basic options' next to the 'manually specify player' checkbox.

**Reply by ddw04** [reply]
![Untitled](https://user-images.githubusercontent.com/77309593/206055335-e4e6e5c1-359b-4e23-b908-fdaed59b7f33.png)
so that kind of worked, but now i got the bar at the top.?  i went to 5 before anything changed and this is the first thing that happened. but it doesnt seem to always happen cause i went up to 7 to see if it fixed the top but nothing happened so i went back down to 5 and it sometimes works and sometimes doesnt. what i mean by that is i could press "w" twice and it works or id need to keep refershing page and pressing "w" multiple times before itll work

**Reply by tamius-han** [reply]
Try repositioning the video by holding shift while moving your mouse over video.

If the black bar remains, you will have to write some custom CSS. At the moment I'm trying to redesign the interface in a way that would allow dealing with this stuff easier, but given the amount of free time I have lately the progress has been glacial.

**Reply by ddw04** [reply]
its not letting me move the video. though i was able to do so with another movie. pretty neat as i never knew that was a thing haha. im not sure why videos like this cant be cropped correctly. seems any video that starts with black bars on all sides cant be stretched,  zoomed, or cropped. as for custom css yeah im not a coder so i have no clue what to do that haha.

thanks for replying though! ill just have to take this off my watch list.
